### ISSUE TITLE
Bug: Favicon is not always visible

### ISSUE THREAD

**Issue by tibequadorian** [ArchiveBox/ArchiveBox#1213]
#### Describe the bug
When archiving any site just using a minimal set of extractors like `--extract title,favicon,readability`, the favicon is **never visible** in the index, and the spinner.gif is shown instead.
By adding another extractor to the list, like `pdf` for example, the favicon is visible again, as shown below.

#### Steps to reproduce
```
archivebox add --extract title,favicon,readability https://www.nytimes.com/2023/08/16/health/abortion-pill-ruling.html
```

#### Screenshots or log output
![image](https://github.com/ArchiveBox/ArchiveBox/assets/9560587/df5bc5b8-a409-4894-9a48-47ca93d72c38)

#### ArchiveBox version

```logs
ArchiveBox v0.6.2
Cpython Linux Linux-6.1.0-11-amd64-x86_64-with-glibc2.28 x86_64
IN_DOCKER=True DEBUG=False IS_TTY=True TZ=UTC SEARCH_BACKEND_ENGINE=ripgrep

[i] Dependency versions:
 √  ARCHIVEBOX_BINARY     v0.6.2          valid     /usr/local/bin/archivebox                                                   
 √  PYTHON_BINARY         v3.9.5          valid     /usr/local/bin/python3.9                                                    
 √  DJANGO_BINARY         v3.1.10         valid     /usr/local/lib/python3.9/site-packages/django/bin/django-admin.py           
 √  CURL_BINARY           v7.64.0         valid     /usr/bin/curl                                                               
 √  WGET_BINARY           v1.20.1         valid     /usr/bin/wget                                                               
 √  NODE_BINARY           v15.14.0        valid     /usr/bin/node                                                               
 √  SINGLEFILE_BINARY     v0.3.16         valid     /node/node_modules/single-file/cli/single-file                              
 √  READABILITY_BINARY    v0.0.2          valid     /node/node_modules/readability-extractor/readability-extractor              
 √  MERCURY_BINARY        v1.0.0          valid     /node/node_modules/@postlight/mercury-parser/cli.js                         
 √  GIT_BINARY            v2.20.1         valid     /usr/bin/git                                                                
 √  YOUTUBEDL_BINARY      v2021.04.26     valid     /usr/local/bin/youtube-dl                                                   
 √  CHROME_BINARY         v90.0.4430.93   valid     /usr/bin/chromium                                                           
 √  RIPGREP_BINARY        v0.10.0         valid     /usr/bin/rg                                                                 

[i] Source-code locations:
 √  PACKAGE_DIR           22 files        valid     /app/archivebox                                                             
 √  TEMPLATES_DIR         3 files         valid     /app/archivebox/templates                                                   
 -  CUSTOM_TEMPLATES_DIR  -               disabled                                                                              

[i] Secrets locations:
 -  CHROME_USER_DATA_DIR  -               disabled                                                                              
 -  COOKIES_FILE          -               disabled                                                                              

[i] Data locations:
 √  OUTPUT_DIR            5 files         valid     /data                                                                       
 √  SOURCES_DIR           14 files        valid     ./sources                                                                   
 √  LOGS_DIR              1 files         valid     ./logs                                                                      
 √  ARCHIVE_DIR           11 files        valid     ./archive                                                                   
 √  CONFIG_FILE           81.0 Bytes      valid     ./ArchiveBox.conf                                                           
 √  SQL_INDEX             212.0 KB        valid     ./index.sqlite3                                                             
```

**Comment by pirate** [comment]
Favicon & title alone are not considered enough for the snapshot to be "archived", which is why it shows the in-progress spinner. We could add readability to the set of things it considers enough for it to qualify as "archived" though.

**Comment by tibequadorian** [comment]
I'm fine with your suggestion as it works out in my case but in general wouldn't it make more sense to show the spinner just as long as the archiving process takes, no matter what extractors are used?

**Comment by pirate** [comment]
Fixed in 19e9c1c2f0a0bb755bf695281271603c44edbdd4, will be out in `v0.7.3`.
